backend/test/game-invite.e2e-spec.ts

```diff
@@ -8,19 +8,26 @@ import { CreateGameInviteDto } from '../src/pong/game_invite/dto/create-game-inv
 import { GameInvitesController } from '../src/pong/game_invite/game-invite.controller';
 import { GameInvite } from '../src/pong/game_invite/entities/game-invite.entity';
 import { GameInvitesModule } from '../src/pong/game_invite/game-invite.module';
+import { UserModule } from '../src/user/user.module';
+import { MatchmakingRequest } from '../src/pong/matchmaking-request/entities/matchmaking-request.entity';
+import { User } from '../src/user/entities/user.entity';
+import { Game } from '../src/pong/game/entities/game.entity';
 
 describe('GameInvite (e2e)', () => {
 	let app: INestApplication;
 	let invitesController: GameInvitesController;
 
-	const mockInvite: CreateGameInviteDto = { source_id: 1, target_id: 2 };
+	const mockInvite: CreateGameInviteDto = {players: [1, 2]};
```

Comment:
not sure but would you need create users to place them into the players? or maybe i'm fully missing the point!!!!

Reply:
Added a test for the creation of the relations :)

Reply:
Was only testing the DTO up to this point